Description: remove duplicate NAME sections, keep the one with a synopsis
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2019-10-25

--- a/lib/Sys/Info/Driver/Linux.pm
+++ b/lib/Sys/Info/Driver/Linux.pm
@@ -29,7 +29,7 @@
 
 =head1 NAME
 
-Sys::Info::Driver::Linux
+Sys::Info::Driver::Linux - Linux driver for Sys::Info
 
 =head1 VERSION
 
@@ -43,10 +43,6 @@
 
 This is the main module in the C<Linux> driver collection.
 
-=head1 NAME
-
-Sys::Info::Driver::Linux - Linux driver for Sys::Info
-
 =head1 METHODS
 
 None.
--- a/lib/Sys/Info/Driver/Linux/Constants.pm
+++ b/lib/Sys/Info/Driver/Linux/Constants.pm
@@ -74,7 +74,7 @@
 
 =head1 NAME
 
-Sys::Info::Driver::Linux::Constants
+Sys::Info::Driver::Linux::Constants - Constants for Linux driver
 
 =head1 VERSION
 
@@ -86,10 +86,6 @@
 
 Constants for Linux driver.
 
-=head1 NAME
-
-Sys::Info::Driver::Linux::Constants - Constants for Linux driver
-
 =head1 METHODS
 
 None.
--- a/lib/Sys/Info/Driver/Linux/Device.pm
+++ b/lib/Sys/Info/Driver/Linux/Device.pm
@@ -13,7 +13,7 @@
 
 =head1 NAME
 
-Sys::Info::Driver::Linux::Device
+Sys::Info::Driver::Linux::Device - Base class for Linux device drivers
 
 =head1 VERSION
 
@@ -27,10 +27,6 @@
 
 Base class for Linux device drivers.
 
-=head1 NAME
-
-Sys::Info::Driver::Linux::Device - Base class for Linux device drivers
-
 =head1 METHODS
 
 None.
--- a/lib/Sys/Info/Driver/Linux/Device/CPU.pm
+++ b/lib/Sys/Info/Driver/Linux/Device/CPU.pm
@@ -94,7 +94,7 @@
 
 =head1 NAME
 
-Sys::Info::Driver::Linux::Device::CPU
+Sys::Info::Driver::Linux::Device::CPU - Linux CPU Device Driver
 
 =head1 VERSION
 
@@ -108,10 +108,6 @@
 
 Identifies the CPU with L<Unix::Processors>, L<POSIX> and C<< /proc >>.
 
-=head1 NAME
-
-Sys::Info::Driver::Linux::Device::CPU - Linux CPU Device Driver
-
 =head1 METHODS
 
 =head2 identify
--- a/lib/Sys/Info/Driver/Linux/OS.pm
+++ b/lib/Sys/Info/Driver/Linux/OS.pm
@@ -263,7 +263,7 @@
 
 =head1 NAME
 
-Sys::Info::Driver::Linux::OS
+Sys::Info::Driver::Linux::OS - Linux backend
 
 =head1 VERSION
 
@@ -277,10 +277,6 @@
 
 -
 
-=head1 NAME
-
-Sys::Info::Driver::Linux::OS - Linux backend
-
 =head1 METHODS
 
 Please see L<Sys::Info::OS> for definitions of these methods and more.
--- a/lib/Sys/Info/Driver/Linux/OS/Distribution.pm
+++ b/lib/Sys/Info/Driver/Linux/OS/Distribution.pm
@@ -360,7 +360,7 @@
 
 =head1 NAME
 
-Sys::Info::Driver::Linux::OS::Distribution
+Sys::Info::Driver::Linux::OS::Distribution - Linux distribution probe
 
 =head1 VERSION
 
@@ -394,10 +394,6 @@
 redflag and ubuntu(lsb). People running unsupported distro's are greatly
 encouraged to submit patches.
 
-=head1 NAME
-
-Sys::Info::Driver::Linux::OS::Distribution - Linux distribution probe
-
 =head1 METHODS
 
 =head2 build
--- a/lib/Sys/Info/Driver/Linux/OS/Distribution/Conf.pm
+++ b/lib/Sys/Info/Driver/Linux/OS/Distribution/Conf.pm
@@ -259,7 +259,7 @@
 
 =head1 NAME
 
-Sys::Info::Driver::Linux::OS::Distribution::Conf
+Sys::Info::Driver::Linux::OS::Distribution::Conf - Distro configuration
 
 =head1 VERSION
 
@@ -269,10 +269,6 @@
 
 =head1 DESCRIPTION
 
-=head1 NAME
-
-Sys::Info::Driver::Linux::OS::Distribution::Conf - Distro configuration
-
 =head1 AUTHOR
 
 Burak Gursoy <burak@cpan.org>
